data/api-impl/src/commonMain/kotlin/io/github/droidkaigi/confsched2019/data/api/KtorDroidKaigiApi.kt

```diff
@@ -27,4 +32,13 @@ open class KtorDroidKaigiApi constructor(
             accept(ContentType.Application.Json)
         }
     }
+
+    override suspend fun getAnnouncements(lang: LangParameter): AnnouncementListResponse {
+        val rawResponse = httpClient.get<String> {
```

Comment:
Kotlinx.serialization does not support deserializing generic types like List. So we need to register the concrete serializer to JsonFeature or parse it manually by JSON.
I used JSON#parse to solve this matter because JSONFeature must know models to be serialized to register their serializers, unfortunately. 😢